pyomo/contrib/mindtpy/extended_cutting_plane.py

```diff
@@ -0,0 +1,256 @@
+
+# -*- coding: utf-8 -*-
+
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright (c) 2008-2022
+#  National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+"""Implementation of the MindtPy solver.
```

Comment:
Should this be included in this file? Shouldn't we rather just include it in the main MindtPy file?

Reply:
This seems to be the main file, but why is it called `extended_cutting_plane.py`?

Reply:
The main file is `MindtPy.py`. This file only includes ECP method. I have removed the version descriptions.